.github/workflows/ci.yml

```diff
@@ -5,67 +5,58 @@ jobs:
   build:
     name: Build and test
     runs-on: ubuntu-latest
+    environment: TodoApp_Environment,Repository secrets
```

Comment:
I think that we changed from using a specific environment to just using the repository level secrets, so we should be able to remove this now

Reply:
Done